src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -32,6 +32,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
     <PackageReference Include="System.Collections.Specialized" Version="4.0.1" />
+    <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
```

Comment:
Why were we not getting any compiler errors when this reference was not present?

Reply:
I believe it was in a separate `ItemGroup`
https://github.com/vincentkam/mongo-csharp-driver/blob/821ba56ebedd982b713f6d995e51244df88d5ffe/src/MongoDB.Driver.Core.Dotnet/MongoDB.Driver.Core.Dotnet.csproj#L22